index.html

```diff
@@ -6,12 +6,193 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Document</title>
+    <script src="https://kit.fontawesome.com/39fd49c23e.js" crossorigin="anonymous"></script>
+
 </head>
 
 <body>
-    <nav class="navbar">
-        <h2>hypnosis</h2>
-    </nav>
+    <div class="nav-content">
+        <nav class="navbar">
+            <button><i class="fa-sharp fa-solid fa-magnifying-glass"></i></button>
+            <h2>hypnosis</h2>
+            <button><i class="fa-sharp fa-solid fa-bars"></i></button>
+        </nav>
+    </div>
+    <fieldset class="content">
+        <header>
+            <h1>Stand up for Somalia</h1>
+            <hr>
+            <img src="christine.png">
+            <h2>Christine Kolesnikov</h2>
+            <p>Hypnosis - Senior Editor</p>
+            <button><a href=""><i class="fa-brands fa-facebook"></i> Shere</a></button>
+            <button><a href=""><i class="fa-brands fa-twitter"></i> Tweet</a></button>
+            <button><a href=""><i class="fa-brands fa-linkedin"></i> Share</a></button>
+            <button><a href=""><i class="fa-sharp fa-solid fa-link"></i> Copy</a></button>
+        </header>
+
+        <hr>
+        <article>
+            <img src="people1.png">
+            <h2>People are dying of hunger...
+                there’s no water’</h2>
+            <p>In Somalia, more than 6 million people are in need of food assistance more than half the population.
+                But even inside the camps, food is scarce.</p>
+            <hr>
+            <img src="let's.png">
+            <h2>Let’s help them. I know you can</h2>
+            <p>Girls, boys and families are experiencing food insecurity, higher food and other commodity prices coupled
```

Comment:
this is great, eny article in the project need the project tag :+1:

Reply:
Thank you for your revision @hendridg I made several branches which I merged to develop there are 5 functions I made 5 pull requests with different branches all of them were merged to deelop, I will take your advice into account to make the corresponding revisions, thanks 👍